src/System.Text.Json/tests/Serialization/Value.ReadTests.cs

```diff
@@ -260,6 +260,11 @@ public static void ValueFail()
             Assert.Throws<JsonReaderException>(() => JsonSerializer.Parse<DateTime?>(unexpectedString));
             Assert.Throws<JsonReaderException>(() => JsonSerializer.Parse<DateTime?>(unexpectedString));
 
+            Assert.Throws<JsonReaderException>(() => JsonSerializer.Parse<DateTimeOffset>(unexpectedString));
+            Assert.Throws<JsonReaderException>(() => JsonSerializer.Parse<DateTimeOffset>(unexpectedString));
+            Assert.Throws<JsonReaderException>(() => JsonSerializer.Parse<DateTimeOffset?>(unexpectedString));
+            Assert.Throws<JsonReaderException>(() => JsonSerializer.Parse<DateTimeOffset?>(unexpectedString));
```

Comment:
@steveharter, what's the rationale for doing 2 calls per type? Is it to validate that repeated calls fail as expected?

Reply:
Please remove the duplicates. I used to have two strings to check; now I just have one.